CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 ## [Unreleased 2.x]
 ### Added
+- QueryGroup resource tracking and cancellation framework changes ([#13897](https://github.com/opensearch-project/OpenSearch/pull/13897))
```

Comment:
This is still not correct. Removing other entries is the not the right way to fix this

Reply:
Actually, my bad. Unreleased 2.x is stripped clean due to 2.16 fork!

Reply:
Yes, also if i have removed all the other lines, it would have appeared under deleted section in Red here